stdlib/gc.mli

```diff
@@ -224,15 +224,17 @@ type control =
 
 external stat : unit -> stat = "caml_gc_stat"
 (** Return the current values of the memory management counters in a
-   [stat] record that represent the program's total memory stats.
-   This function causes a full major collection. *)
+    [stat] record that represents the program's total memory stats.
+    The [heap_chunks], [free_blocks], [largest_free], and [stack_size] metrics
+    are currently not available in OCaml 5: their returned field values are
+    therefore [0].
+    This function causes a full major collection. *)
 
 external quick_stat : unit -> stat = "caml_gc_quick_stat"
-(** Same as [stat] except that [live_words], [live_blocks], [free_words],
-    [free_blocks], [largest_free], and [fragments] are set to 0. Due to
-    per-domain buffers it may only represent the state of the program's
-    total memory usage since the last minor collection or major cycle.
-    This function is much faster than [stat] because it does not need to
+(** Returns a record with the current values of the memory management counters
+    like [stat]. Due to per-domain buffers it may only represent the state of
+    the program's total memory usage since the last minor collection or major
```

Comment:
Are we sure this should be “since”? It does not read right to me. The number of minor GCs since the last minor GC will be 0, but what quick_stat returns is different. How about:

```
(** Returns a record with the current values of the memory management 
  counters like [stat]. Unlike [stat], [quick_stat] does not perform a full major 
  collection, and hence, is much faster. However, [quick_stat] reports the 
  counters sampled at the last minor collection or at the end of the last major 
  collection cycle. Hence, the memory stats returned by [quick_stat] are not 
  instantaneously accurate. *)
```

Reply:
Also, in the original comment, I am not sure which field `the program's total memory usage` is supposed to represent (if at all). The documentation attached to `stats` refers to `the program's total memory stats`, while being vague, is a bit more accurate. I did wonder whether `usage` was supposed to be `stats`.

Reply:
Sounds good to me!
The reordering of sentences also improve readability IMO, mentioning the central difference first and only secondly going into more detail.